test/protocol/FundsHandlerTest.js

```diff
@@ -367,12 +367,15 @@ describe("IBosonFundsHandler", function () {
 
           await foreign20ReturnFalse.connect(assistant).mint(await assistant.getAddress(), depositAmount);
           await foreign20ReturnFalse.connect(assistant).approve(protocolDiamondAddress, depositAmount);
+          const safeERC20 = await ethers.getContractAt("SafeERC20", protocolDiamondAddress);
 
           await expect(
             fundsHandler
               .connect(assistant)
               .depositFunds(seller.id, await foreign20ReturnFalse.getAddress(), depositAmount)
-          ).to.revertedWith(RevertReasons.SAFE_ERC20_NOT_SUCCEEDED);
+          )
+            .to.revertedWithCustomError(safeERC20, "SafeERC20FailedOperation")
```

Comment:
I suppose we should also put custom errors in the revert-reasons.js for consistency.